### PR TITLE
Remove thumb creation

### PR DESCRIPTION
**Pull request by poupotte**
Remove thumb creation because it's managed by DS (refs https://github.com/cozy/cozy-data-system/pull/127)

### CODE REVIEW

**Comment by frankrousseau** [comment]
It lacks of comments.

File: server/controllers/file.coffee [hunk 1]
```diff
@@ -85,21 +85,25 @@ module.exports.createPhoto = (req, res, next) ->
...
+        else
+            # Add content thumb or screen if necessary
+            rawFile = "/tmp/#{photo.id}"
+            fs.openSync rawFile, 'w'
+            stream = file.getBinary 'file', (err) ->
+                return next err if err
+            stream.pipe fs.createWriteStream rawFile
+            stream.on 'error', next
+
+            stream.on 'end', =>
```
Comment (frankrousseau) [inline server/controllers/file.coffee:103]: It looks like the `res.on 'close'` is not handled. When the user cancels his requests, streams should be aborted.

**Comment by jsilvestre** [comment]
@frankrousseau @aenario  this PR is _very very_ important, we currently have a **big major** bug that is not just annoying, but people who have it will have to perform actions manually (rm -rf /tmp/).

**Comment by jsilvestre** [comment]
Thank you!

**Pull request merged**